app/src/main/java/com/github/palFinderTeam/palfinder/meetups/activities/MeetupListActivity.kt

```diff
@@ -87,26 +105,56 @@ class MeetupListActivity : AppCompatActivity() {
         }
     }
 
-    fun sortByCap(view: View?) {
+    fun sortByCap() {
```

Comment:
Might be better to put the sorting and filter method inside a viewmodel class

Reply:
the problem is that the filtering is located inside the adapter class, and I cannot link the viewModel class with the Adapter one